internal/testutil/context.go

```diff
@@ -34,6 +34,9 @@ func NewTestCtx(t *testing.T, role auth.Role, projects ...int64) TestCtx {
 		Project: project,
 	})
 
+	log := zaptest.NewLogger(t)
+	zap.ReplaceGlobals(log)
```

Comment:
This makes a zap logger that runs in tests, but because it's a `zaptest.Logger`, it doesn't log unless you run `go test -v`. Is nice.